### PR TITLE
Renames charcoal bottles

### PR DESCRIPTION
**Pull request by ghost**
:cl: Mark9013100
tweak: Charcoal bottles have been renamed, and have been given a more informative description.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Decided to rename the charcoal bottles, as there's already a reagent called antitoxin in the code. Also modifies the description to be more informative of it's purpose.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 1]
```diff
@@ -73,8 +73,8 @@
 	list_reagents = list("chloralhydrate" = 15)
 
 /obj/item/reagent_containers/glass/bottle/charcoal
-	name = "antitoxin bottle"
-	desc = "A small bottle of charcoal."
+	name = "charcoal (antitoxin) bottle"
```
Comment (lzimann) [inline code/modules/reagents/reagent_containers/bottle.dm]: leaving the (antitoxin) makes it a bit confusing imo

Comment (Cyberboss) [reply code/modules/reagents/reagent_containers/bottle.dm]: Yeah, basically every player knows what charcoal does and the new ones will learn

Comment (Mark9013100) [reply code/modules/reagents/reagent_containers/bottle.dm]: I'll gut the antitoxin part then

**Review by ActualRory — APPROVED** [review]

**Comment by jughu** [comment]
Anti-toxins is called that way to be easy found as it says anti toxins means it removes toxins and charcoal is just ???

**Comment by ActualRory** [comment]
Makes sense to me.

**Comment by lzimann** [comment]
The problem is that antitoxin is a different reagent, so, imagine having a recipe that requires antitoxin, the person would use that bottle thinking it was antitoxin(when in reality it contains charcoal)

**Comment by jughu** [comment]
why the fuck would anyone make anti toxins when charcoal is easier to make

**Comment by Mark9013100** [comment]
anti-toxins is easier to make, as you can make in like a second from the chem dispenser, where else charcoal requires the use of the heater.

**Comment by DaxDupont** [comment]
Also antitox is in the sleepers.
It's important to make the distinction since 20(+)u of antitoxin will OD you but you can gobble charcoal ad infinitum.

**Comment by lzimann** [comment]
we really should merge both chemicals or add more distinction between both or merge the chemicals already

**Comment by Wyzack** [comment]
Calling it charcoal has always been pants-on-head turbotarded. If you injected any kind of charcoal int your blood you would fucking die. It was better with a generic sci-fi name

**Comment by jughu** [comment]
Tbh just remove charcoal and keep anti toxins

**Comment by jughu** [comment]
I know charcoal purges chem but we have 2 other chems for that

**Comment by TribeOfBeavers** [comment]
@Wyzack

Activated charcoal is a thing used to treat poisons IRL.

You have to injest it though, injecting is probably a bad idea (does ss13 differentiate between those?).

**Comment by Doppeltongue** [comment]
It differentiates between touch, ingest, vapor, inject, and patch.

**Comment by ExcessiveUseOfCobblestone** [comment]
but not currently

**Review by Cyberboss — APPROVED** [review]

**Comment by Bawhoppen** [comment]
Anti-toxin has a real name and it's Dyvolene or something like that

**Comment by Mark9013100** [comment]
That's what baycode and it's derivatives have it named as.

**Pull request merged**